be/src/io/fs/azure_obj_storage_client.cpp

```diff
@@ -75,6 +78,40 @@ ObjectStorageResponse do_azure_client_call(Func f, const ObjectStoragePathOption
     return {};
 }
 
+struct AzureBatchDeleter {
+public:
+    AzureBatchDeleter(Azure::Storage::Blobs::BlobContainerClient* client,
+                      const ObjectStoragePathOptions& opts)
+            : _client(client), _batch(client->CreateBatch()), _opts(opts) {}
+    void delete_blob(const std::string& blob_name) {
+        deferred_resps.emplace_back(_batch.DeleteBlob(blob_name));
+    }
+    ObjectStorageResponse execute() {
+        auto resp = do_azure_client_call([&]() { _client->SubmitBatch(_batch); }, _opts);
```

Comment:
Is there any limit for this _batch, say is it ok that  10000000 blobs to delete?